File: cilicili-cloud/cilicili-account/src/main/java/com/gaoyuanming/account/model/entity/Account.java
package com.gaoyuanming.account.model.entity;

import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Data
@Entity
@Table(name = "t_account")
@NoArgsConstructor
public class Account {

    //用户的微信openId
    @Id
    private String id;

    private String nickname;

    private String address;

    private String gender;

    private String avatarUrl;

    //是否获取过微信的user_info
    private boolean hasInfo;

    private int fansCount;

    private int followCount;

}

File: cilicili-go/model/entity/account.go
package entity

type User struct {
	Id          string `json:"id"`
	Nickname    string `json:"nickname"`
	Address     string `json:"address"`
	Gender      string `json:"gender"`
	AvatarUrl   string `json:"avatarUrl"`
	HasInfo     bool   `json:"hasInfo"`
	FansCount   int    `json:"fansCount"`
	FollowCount int    `json:"followCount"`
}

type Admin struct {
	Id       string `json:"id"`
	Email    string `json:"email"`
	Password string `json:"password"`
}

type Follow struct {
	Id         string `json:"id"`
	FromUserId string `json:"fromUserId" gorm:"form_user_id"` //谁关注的 user_id
	ToUserId   string `json:"toUserId" gorm:"to_user_id"`     //关注的谁 user_id
}
